src/components/common/Skeleton/index.tsx

```diff
@@ -0,0 +1,29 @@
+import $ from './skeleton.module.scss';
+
+interface SkeletonProps {
+  width?: string;
+  height?: string;
+  borderRadius?: string;
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+export default function Skeleton({
+  width,
+  height,
+  borderRadius = '4px',
+  className,
+  style,
+}: SkeletonProps) {
+  return (
+    <div
+      className={`${$.skeletonBase} ${className}`}
```

Comment:
classnames 사용하면 두 클래스를 묶어줘서 더 가독성이 좋을거같아요!
```suggestion
      className={classnames($.skeletonBase, $className)}
```

Reply:
네~! 그렇게 수정해서 다시 올릴게요!